CustomList task

FranzVidangosPari opened the pull request:
Metods:
   void add(int value);
    void printList();
    void removeFirst();
    void addFirst(int value);
    void addLast(int value);
    void addIndex(int index,int element);
    void removeIndex(int index);
    void removeLast();

FranzVidangosPari committed:
CustomList task

Metods:
   void add(int value);
    void printList();
    void removeFirst();
    void addFirst(int value);
    void addLast(int value);
    void addIndex(int index,int element);
    void removeIndex(int index);
    void removeLast();

jpsandovaln commented on homework-1/src/CustomList.java:
```diff
@@ -0,0 +1,123 @@
+public class CustomList implements IList {
```
Add documentations

jpsandovaln commented on homework-1/src/Node.java:
```diff
@@ -0,0 +1,11 @@
+    Node(int value){
+        this.value = value;
+        this.next = null;
+
```
remove extra line

FranzVidangosPari committed:
generic CustomList, queue and stack

I update the CustomList to generic, and create customQueue and CustomStack classes

jpsandovaln commented:
I merged it but you need to add unit tests
